test/app-media-recorder.html

```diff
@@ -62,168 +57,168 @@
     </test-fixture>
 
     <script type="module">
-import './helpers-import.js';
-import '../app-media-devices.js';
-import '../app-media-stream.js';
-import '../app-media-recorder.js';
-suite('app-media-recorder', function() {
-  var awaitEvent = AppMediaTestHelpers.awaitEvent;
-  var timePasses = AppMediaTestHelpers.timePasses;
-  var createDevice = AppMediaTestHelpers.createDevice;
-  var setDevices = AppMediaTestHelpers.setDevices;
-
-  suiteSetup(function() {
-    var fakeDevices = [createDevice('videoinput')];
-    setDevices(fakeDevices);
-    return window.ensureWebRtcPolyfill.then(loadScript('./helpers.js'));
-  });
-
-  setup(function() {
-    if (!window.MediaRecorder) {
-      this.skip();
-    }
-    AppMediaTestHelpers.setRecorderData(new Blob(new Uint8Array(1)));
-  });
-
-  teardown(function() {
-    AppMediaTestHelpers.restoreRecorderData();
-  });
-
-  suite('with basic configuration', function() {
-    var element;
-    var stream;
-
-    setup(function() {
-      if (!window.MediaRecorder) {
-        this.skip();
-      }
-      var recorder = fixture('basic');
-      element = recorder.$.recorder;
-      stream = recorder.$.stream;
-
-      return awaitEvent(element, 'recorder-changed');
-    });
-
-    test('can record from the stream', function() {
-      var dataChanges = awaitEvent(element, 'data-changed');
-
-      element.start();
-
-      timePasses(1).then(function() {
-        element.stop();
-      });
-
-      return dataChanges.then(function() {
-        expect(element.data).to.be.instanceof(Blob);
-      });
-    });
-
-    test('can record using the recording property', function() {
-      var dataChanges = awaitEvent(element, 'data-changed');
-
-      element.recording = true;
-
-      timePasses().then(function() {
-        element.recording = false;
-      });
-
-      return dataChanges.then(function() {
-        expect(element.data).to.be.instanceof(Blob);
-      });
-    });
+      import './helpers-import.js';
+      import '../app-media-devices.js';
+      import '../app-media-stream.js';
+      import '../app-media-recorder.js';
```

Comment:
These are already included above.

Reply:
maybe I should just put them in one big script tag, because modules run async